README.md

```diff
@@ -22,7 +23,7 @@ dependencies. The one downside is that you must first install Docker!
 
 ### First run
 The first time you run Splits I/O with Docker is the best, because you'll have time to get a coffee! Yum! After the
-first run, it will be much quicker.
+first run, it will be much quicker (on Linux, docker commands may need to be prefixed with sudo).
```

Comment:
What is root needed for? I'd prefer to not require users to need it, so if there's something else we can do to let people run docker as themselves, we should probably do it.

Reply:
Docker attaches itself to a unix socket, owned by root so in order to talk to it you need to sudo it.  The alternative is to create a docker group and add your user to that, but that docker group is essentially a root priviledged group.
https://docs.docker.com/engine/installation/linux/linux-postinstall/

Reply:
Gotcha, ok. I wonder why this isn't required on Mac, but the README change looks fine given that situation.